src/main/scala/org/apache/spark/sql/execution/datasources/v2/merge/parquet/batch/merge_operator/MergeOperator.scala

```diff
@@ -52,8 +52,15 @@ trait MergeOperator[T] extends Serializable {
 }
 
 class DefaultMergeOp[T] extends MergeOperator[T] {
+  override def mergeData(input: Seq[T]): T = {
+    input.filter(!_.equals("null")).last
+  }
+}
+
+class MergeNullUpdate[T] extends MergeOperator[T] {
```

Comment:
Would it be better if we have a global option to choose a default between these two merge operator